wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireOutput.kt

```diff
@@ -31,7 +31,7 @@ import javax.inject.Inject
  * as destination directories and configuration options). This includes registering output
  * directories with the project so they can be compiled after they are generated.
  */
-abstract class WireOutput {
+sealed class WireOutput {
```

Comment:
Is this required for your PR to pass? If not, could you remove all collateral changes to make the review easier please?